deps/dns.lua

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 -- https://github.com/openresty/lua-resty-dns/blob/master/lib/resty/dns/resolver.lua
 
 exports.name = "luvit/dns"
-exports.version = "1.0.2"
+exports.version = "1.0.3"
```

Comment:
Since this has added functionality, but is still backwards compatible, we should bump the middle version number